project/frontend/src/containers/Competition/Competition.js

```diff
@@ -44,47 +44,45 @@ const Competition = (props) => {
         return formatDHM(toDayHourMinute(remainingTime));
     };
 
-    const placeholderCompetitors = [
-        {
-            rank : 1,
-            rankYesterday : 2,
-            name : "emmahogan@",
-            available : 100,
-            net_worth : 250,
-            num_searches : 5,
-        },
-        {
-            rank : 2,
-            rankYesterday : 1,
-            name : "mercurylin@",
-            available : 55,
-            net_worth : 220,
-            num_searches : 4,
-        },
-        {
-            rank : 3,
-            rankYesterday : 3,
-            name : "phoebek@",
-            available : 120,
-            net_worth : 120,
-            num_searches : 0,
-        },
-    ];
+    const usernameFromEmail = (email) => {
+        var username = email.split('@')[0] + '@';
+        return username;
+    }
+
+    const formatChartData = () => {
+        const networths = props.networths;
+        var data = [];
+        var first_row = ["x"];
+        // add name row to data
+        for (var i = 0; i < networths.length; i++) {
+            var email = networths[i].competitorEmail;
+            first_row.push(usernameFromEmail(email));
+        }
+        data.push(first_row);
+
+        // add networth data
+        const numDays = networths[0].dataPoints.length;
+        for (i = 0; i < numDays; i++) {
+            var row = [i];
+            for (var j = 0; j < networths.length; j++) {
+                row.push(networths[j].dataPoints[i]);
+            }
+            data.push(row);
+        }
+        return data;
+    }
+
+    const competitors = props.rankings;
+    const generalInfo = props.generalInfo;
```

Comment:
There is inconsistent use of these variables later on, might be worth just referencing `props` directly

Reply:
done